04/app.js

```diff
@@ -1,69 +1,128 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
+import searchuser from './components/styles/searchuser';
+import oddnumberuser from './components/styles/oddnumberuser';
+import evennumberuser from './components/styles/evennumberuser';
+
 class App extends React.Component {
     state = { 
         firstName: '',
         lastName: '',
         searchQuery: '',
-        users: ['Jan Kowalski', 'Michał Nowak'],
+        users: ['Jan Kowalski',
+                'Michał Nowak',
+                'Żaneta Żatecka',
+                'Ewelina Mróz',
+                'Maciej Cebulowski',
+                'Sandra Stasiak'],
     }
 
-    renderUsersList() {
-        const {users} = this.state;
-        return users.map(name => {
-            return (
-                <li onClick={ this.clickHandler }>
-                    { name }
-                </li>
-            );
+    renderUsersList = () => {
+        const {users, searchQuery} = this.state;
+
+        // alfabetyczne sortowanie users
+        users.sort((a, b) => {
+            return (a < b) ? -1 : ((a > b) ? 1 : 0);
+        });
+
+        // jeśli wyszukaj użytkownika jest puste to renderuj na stronę całą listę użytkowników
+        let i = -1;
+        let j = 0;
+        if (searchQuery === '') {
+            return users.map(name => {
```

Comment:
W `map()` drugi parametr to `index` i możesz go wykorzystać zamiast robić dodatkową zmienną `i` czy `j`